sdk/communication/Azure.Communication.CallingServer/samples/Sample1_CreateCall.md

```diff
@@ -27,12 +27,12 @@ var createCallOption = new CreateCallOptions(
        });
 ```
 ```C# Snippet:Azure_Communication_Call_Tests_CreateCall
-CreateCallResponse createCallResponse = callClient.CreateCall(
+var callConnection = callingServerClient.CreateCallConnection(
     source: new CommunicationUserIdentifier("<source-identifier>"), // Your Azure Communication Resource Guid Id used to make a Call
     targets: new List<CommunicationIdentifier>() { new PhoneNumberIdentifier("<targets-phone-number>") }, // E.164 formatted recipient phone number
     options: createCallOption // The options for creating a call.
     );
-Console.WriteLine($"Call Leg id: {createCallResponse.CallLegId}");
+Console.WriteLine($"Call connection id: {callConnection.Value.CallConnectionId}");
```

Comment:
>.Value [](http://example.com/codeflow?start=55&length=6)

why do we need .Value?

Reply:
Apis are returning Response object which kind of wrapped http response object. The Value property contain the return value.

Reply:
I believe you can directly just get `callConnection.CallConnectionId` Can you give it a try?

Reply:
then variable should be named differently - something like callConnectionResponse